docs/source/_toctree.yml

```diff
@@ -0,0 +1,24 @@
+- sections:
```

Comment:
I think a `title` should also be added here (like below with `title: "Reference`).

![image](https://user-images.githubusercontent.com/342922/161014568-2e9184f5-bfac-4f5d-a445-1d6b32be8c5b.png)

https://moon-ci-docs.huggingface.co/docs/huggingface_hub/pr_782/en/index

Reply:
Indeed, good catch! Will add this.